.tekton/controller-push.yaml

```diff
@@ -23,15 +23,9 @@ spec:
         sed -i 
         -e 's|\(https://github.com/redhat-appstudio/jvm-build-service/.*?ref=\)\(.*\)|\1{{ revision }}|'
         -e 's/\(newTag: \).*/\1{{ revision }}/'
-        -e 's|\(https://raw.githubusercontent.com/redhat-appstudio/jvm-build-service/\)[^/]*\(/.*\)|\1{{ revision }}\2|'
-        -e 's|\(quay.io/redhat-appstudio/[^:]*:\).*|\1{{ revision }}|'
-        components/build/jvm-build-service/kustomization.yaml &&
-        sed -i 
-        -e 's|\(quay.io/redhat-appstudio/[^:]*:\).*|\1{{ revision }}|'
-        components/build/jvm-build-service/operator-images.yaml&&
-        sed -i 
-        -e 's|\(quay.io/redhat-appstudio/[^:]*:\).*|\1{{ revision }}|'
-        components/build/jvm-build-service/system-config.yaml
+        components/hacbs/jvm-build-service/base/kustomization.yaml
+
+        ./hack/generate-apiexport-overlays.sh components/hacbs/jvm-build-service/
```

Comment:
where does `generate-apiexport-overlays.sh` live @Michkov ?

I don't see it in the jvm-build-service main branch.

Reply:
this is working inside infra-deployments (same as the componetns/hacbs/jvm-build-service)